linera-chain/src/chain.rs

```diff
@@ -1235,5 +1241,5 @@ fn empty_executed_block_size() {
         outcome: crate::data_types::BlockExecutionOutcome::default(),
     };
     let size = bcs::serialized_size(&executed_block).unwrap();
-    assert_eq!(size, EMPTY_EXECUTED_BLOCK_SIZE);
+    assert_eq!(size, EMPTY_BLOCK_SIZE);
```

Comment:
TODO: Test for empty block (new) size.

Reply:
Done in [Testy empty Block size.](https://github.com/linera-io/linera-protocol/pull/3101/commits/5fe042587db04596109fa57d5d66cb29303122bc)

Reply:
You're right - it doesn't make sense to keep this test for the `ExecutedBlock` type.